### PR TITLE
Feature/mar-31 edit profile

### PR DESCRIPTION
**Pull request by Artem-H5**

### CODE REVIEW

File: src/components/pages/ProfilePage/ProfilePage.tsx
Comment (atiragramk) [inline src/components/pages/ProfilePage/ProfilePage.tsx:1]: font size for most used headers in app is already specified in theme.

Comment (atiragramk) [inline src/components/pages/ProfilePage/ProfilePage.tsx:1]: font size for most used headers in app is already specified in theme.

`                    <Typography
                        variant="h4"
                        fontSize="1rem"
                        fontWeight="700"
                    >Ольга</Typography>`
there couldn't be the h4 header according to design

File: src/components/pages/ProfilePage/SettingsTab/style.ts
Comment (atiragramk) [inline src/components/pages/ProfilePage/SettingsTab/style.ts:1]: `export const StyledStar = styled("span")(({ theme }) => ({
    color: "red",
    paddingLeft: theme.spacing(1)
}));` 
for color "red" there is palette 'error' in theme which contains  colors from guideline

File: src/components/pages/ProfilePage/style.ts
Comment (atiragramk) [inline src/components/pages/ProfilePage/style.ts:1]: "#FFF" and #F3F3F3" colors also specified in theme

File: src/components/pages/ProfilePage/SettingsTab/SettingsTab.tsx
Comment (atiragramk) [inline src/components/pages/ProfilePage/SettingsTab/SettingsTab.tsx:1]: `function a11yProps(index: number) {
    return {
        id: `profile-tab-${index}`,
        'aria-controls': `profile-tabpanel-${index}`,
    };
}`
unnecessary function for generation only 2 ids

**Review by atiragramk — APPROVED** [review]

**Review by AnnaYukhno — APPROVED** [review]

**Pull request merged**